### PR TITLE
The Queen is now called the Queen Consort. A coronated Queen Consort/Princess/other female becomes the Queen

### PR DESCRIPTION
**Pull request by BlueMemesauce**
## About The Pull Request

This is mostly just a bug fix, before coronating a new king assigned king the ex-king role which literally doesnt exist and broke the game and made the ex-king runtime everything. So I added an actual ex-king role.

It also changes a few names to sound cooler
* changing the name of roundstart Queen to be more Queen Consort, which is more accurate as she's not a ruling queen, just married to the king.
* an abdicated king is called an King Emeritus instead of Ex-King
* the queen consort of an abdicated king is called a Queen Dowager

## Why It's Good For The Game

the queen name bothered me also coronating people shouldn't break the game

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/church/priest.dm [hunk 1]
```diff
@@ -8,7 +8,6 @@
 	total_positions = 1
 	spawn_positions = 1
 	selection_color = JCOLOR_CHURCH
-	f_title = "Priestess"
```
Comment (BlueMemesauce) [inline code/modules/jobs/job_types/roguetown/church/priest.dm:11]: this was here twice

**Comment by BlueMemesauce** [comment]
need to test this shit more

File: code/controllers/subsystem/rogue/treasury.dm [hunk 2]
```diff
@@ -80,7 +80,7 @@ SUBSYSTEM_DEF(treasury)
 		var/people_told = 0
 		for(var/mob/living/carbon/human/X in GLOB.human_list)
 			switch(X.job)
-				if("King", "Queen", "Steward", "Clerk")
```
Comment (BlueMemesauce) [inline code/controllers/subsystem/rogue/treasury.dm:83]: queen consort doesnt need to be told about the treasury

**Comment by BigBigga** [comment]
i guess that's fair

**Comment by BlueMemesauce** [comment]
ok tested it and it works

image
image

**Comment by TheManWithNoHands** [comment]
Based. Matriarchy is fun. And if this is a RP reward there is no ground to complain about it.

**Comment by BlueMemesauce** [comment]
You could already declare people the queen, this just makes the system better. before it assigned king the ex-king role which literally doesnt exist and broke the game
So this is mostly just a bug fix along with changing the name of Queen to be more historically accurate as she's not a ruling queen, just married to the king.

**Comment by TheManWithNoHands** [comment]
> This is mostly just a bug fix along with changing the name of roundstart Queen to be more historically accurate as she's not a ruling queen, just married to the king.

Doesnt make it less based.

**Comment by Real-MAGNUM** [comment]
Let me know when it's ready for testmerge.

**Comment by BlueMemesauce** [comment]
It's ready, I tested it and it works

**Comment by Real-MAGNUM** [comment]
Testmerging

**Pull request merged**